reference/src/glossary.md

```diff
@@ -207,6 +208,8 @@ requirement of 2.
 
 *Padding* (of a type `T`) refers to the space that the compiler leaves between fields of a struct or enum variant to satisfy alignment requirements, and before/after variants of a union or enum to make all variants equally sized.
 
+Padding for a type is either [interior padding], which is part of one or more fields, or [exterior padding], which is before, between, or after the fields.
```

Comment:
What is this useful for?

Reply:
Unclear if you mean the sentence or the definitions in general.

The definitions of interior and exterior padding I added early on, expecting to need to refer to them, but revisions may have made them less necessary. I will make sure to take a look on whether they are still needed.

This particular sentence was because I find "X is either Y or Z" type statements helpful to in the context of a definition of X, to make clear that all X is exactly one of Y or Z. But if you don't like it I don't mind removing it.

Reply:
I was referring to the distinction of interior and exterior padding... as my later comments indicated, I think that is terminology we just don't need.